server/aws-lsp-codewhisperer/src/language-server/netTransform/validation.ts

```diff
@@ -10,11 +11,20 @@ export function isSolution(userInputrequest: StartTransformRequest): boolean {
     return userInputrequest.SelectedProjectPath.endsWith('.sln')
 }
 
-export function validateProject(userInputrequest: StartTransformRequest): boolean {
+export function validateProject(userInputrequest: StartTransformRequest, logging: Logging): boolean {
     var selectedProject = userInputrequest.ProjectMetadata.find(
         project => project.ProjectPath == userInputrequest.SelectedProjectPath
     )
-    if (selectedProject) return supportedProjects.includes(selectedProject?.ProjectType)
+
+    if (selectedProject) {
+        var isValid = supportedProjects.includes(selectedProject?.ProjectType)
```

Comment:
nit: this should be a const instead of var and then the function could be rewritten like:

```
if (!selectedProject)
   return false
const isValid = supportedProjects.includes(selectedProject.ProjectType)
logging.log(...)
return isValid
```